pages/blog/blog-posts.css

```diff
@@ -1,3 +1,8 @@
+.blog-post-wrapper {
+    position: relative;
+    top: 90px;
+}
+
```

Comment:
Here I have added position 'relative' and moved our blog post content down for 90px (from the top of the page) so that it doesn't get hidden by navbar component 😄